ESEWA MARKET/Home/View/HomeViewController.swift

```diff
@@ -0,0 +1,278 @@
+            let likeButton = UIButton()
+            likeButton.translatesAutoresizingMaskIntoConstraints = false
+            likeButton.tintColor = .red
+            likeButton.setImage(UIImage(systemName: "heart"), for: .normal)
+            return likeButton
+        }()
+        
+        lazy var menuButton: UIButton = {
+            let menuButton = UIButton()
+            menuButton.translatesAutoresizingMaskIntoConstraints = false
+            menuButton.tintColor = .blue
+            menuButton.setImage(UIImage(systemName: "ellipsis"), for: .normal)
+            return menuButton
+        }()
+        
+        cartButton.addTarget(self, action: #selector(didTap), for: .touchUpInside)
+    
+        view.addSubview(footerView)
+        footerView.addSubview(homeButton)
+        footerView.addSubview(cartButton)
+        footerView.addSubview(likeButton)
+        footerView.addSubview(menuButton)
+        
+        NSLayoutConstraint.activate([
+            footerView.topAnchor.constraint(equalTo: view.bottomAnchor),
+            footerView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            footerView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            footerView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -12),
+            footerView.heightAnchor.constraint(equalToConstant: 80),
+            
+            homeButton.centerYAnchor.constraint(equalTo: footerView.centerYAnchor),
+            homeButton.leadingAnchor.constraint(equalTo: footerView.leadingAnchor, constant: 50),
+            
+            cartButton.centerYAnchor.constraint(equalTo: footerView.centerYAnchor),
+            cartButton.leadingAnchor.constraint(equalTo: homeButton.leadingAnchor, constant: 90),
+            
+            likeButton.centerYAnchor.constraint(equalTo: footerView.centerYAnchor),
+            likeButton.leadingAnchor.constraint(equalTo: cartButton.leadingAnchor, constant: 90),
+            
+            menuButton.centerYAnchor.constraint(equalTo: footerView.centerYAnchor),
+            menuButton.trailingAnchor.constraint(equalTo: footerView.trailingAnchor, constant: -50),
+        ])
+    }
+    
+    @objc func didTap() {
+        let vc = CartViewController()
+        vc.navTitle = "My Cart"
+        self.navigationController?.pushViewController(vc, animated: true)
+    }
+}
+
+extension HomeViewController: UITableViewDataSource {
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 7
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
+    }
+    
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        switch section {
+        case 0:
+            return ""
+        case 1:
+            return "Categories"
+        case 2:
+            return "Featured Products"
+        case 3:
+            return "Hot Deals Of TheDay"
```

Comment:
create separate struct which will contain all static texts

Reply:
Implemented the suggested comments and created two separate struct in HomeModel.